app/views/layout-payment-links-v2.njk

```diff
@@ -0,0 +1,81 @@
+{% extends "govuk/template.njk" %}
```

Comment:
I think we'll still need to a keep a version of the layout after switching to the new payment links journey where the service name is in the header, for demo payments at least.
Possibly this layout template could extend the other one but replace the service name with the product name, using blocks maybe? Then we won't end up with repeated code.

Reply:
Thanks - I have now set up `layout.njk` as the base template.
Then I have 2 child templates:
- layout-payment-links-v1.njk - old files will extend this.
- layout-payment-links-v2.njk - new files will extend this.

Reply:
Have updated this - now we just have one new layout for the new Payment Link pages: layout-payment-links-v2.njk
This layout overrides the header and uses the payment link name in the header.